index.js

```diff
@@ -0,0 +1,9 @@
+const divide = function(a, b) {
+    return a / b;
+};
```

Comment:
You can make this a one liner, too!
```suggestion
const divide = (a, b) => a / b;
```